docs/developers/documentation.md

```diff
@@ -0,0 +1,29 @@
+# Building the jsPsych documentation
+
+The documentation hosted at [https://www.jspsych.org](https://www.jspsych.org) is generated using [mkdocs](https://www.mkdocs.org/) and the [mkdocs-material theme](https://squidfunk.github.io/mkdocs-material/). The documentation files are located in the [`/docs` directory of the GitHub repository](https://github.com/jspsych/jsPsych/tree/main/docs). The documentation is written using markdown.
+
+To build a local copy of the docs, you will need to install `mkdocs`, `mkdocs-material`, and `mike` using `poetry`. 
+
+## Install poetry
+
+[Poetry](https://python-poetry.org/) is a package manager for python. Follow the install instructions on the `poetry` website to get it running.
+
+## Install dev dependencies
+
+Run the command `poetry install` in the root directory of jsPsych to install `mkdocs`, `mkdocs-material`, and their dependencies.
+
+## Launch a poetry shell
+
+Poetry installs dependencies in a virtual environment. Run `poetry shell` to open a new shell configured to use the virtual environment that poetry generated.
```

Comment:
Maybe skip this step and recommend `poetry run mike deploy -u [version]` and `poetry run mike serve` instead?

Reply:
nice. made this change!